camille/util/resample.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+
+
+_pandas_supported_interps = [
+    'linear',
+    'time',
+    'index',
+    'values',
+    'nearest',
+    'zero',
+    'slinear',
+    'quadratic',
+    'cubic',
+    'barycentric',
+    'krogh',
+    'polynomial',
+    'spline',
+    'piecewise_polynomial',
+    'from_derivatives',
+    'pchip',
+    'akima',
+]
+
+
+def rm_dupl_indices(ts):
+    return ts[~ts.index.duplicated(keep='first')]
+
+
+def resample(tseries, onto=None, interp='linear'):
+    try:
+        idx = onto.index
+    except:
+        idx = onto
+
+    ts = (
+        pd.concat([tseries, pd.Series(index=idx)])
+        .sort_index(kind='mergesort')
```

Comment:
Why not quicksort(default)?

Reply:
The sort must be stable. When we remove duplicate indices (before reindexing). We keep the first occurance of data, that should come from the function we are resampling, which is first in the series before the sorting